app/views/provider/admin/dashboards/_services_filter.html.slim

```diff
@@ -6,11 +6,14 @@
         var allApis = #{json @services}
 
         var displayApis = function(filteredApis) {
-          for (api = 0; api < allApis.length; api++) {
-            document.getElementById("service_" + allApis[api].service.id).style.display = "none"
-          }
-          for (api = 0; api < filteredApis.length; api++) {
-            document.getElementById("service_" + filteredApis[api].service.id).style.display = "inline"
+          for (let i = 0; i < allApis.length; i++) {
```

Comment:
Why not`Array.prototype.forEach` ?

Reply:
You're right. I took for granted it wasn't supported in IE.